Docs/source/developers/faq.rst

```diff
@@ -0,0 +1,62 @@
+.. _development-faq:
+
+FAQ
+===
+
+This section lists frequently asked developer questions.
+
+
+What is ``0.0_rt``?
+-------------------
+
+It's a C++ `floating-point literal <https://en.cppreference.com/w/cpp/language/floating_literal>`__ for zero of type ``amrex::Real``.
+
+We use `literals <https://en.cppreference.com/w/cpp/language/expressions#Literals>`__ to define constants with a specific type, in that case the zero-value.
+There is also ``0.0_prt``, which is a literal zero of type ``amrex::ParticleReal``.
+In std C++, you know: ``0.0`` (literal ``double``), ``0.0f`` (literal ``float``) and ``0.0L`` (literal ``long double``).
+We do not use use those, so that we can configure floating point precision at compile time and use different precision for fields (``amrex::Real``) and particles (``amrex::ParticleReal``).
+
+You can also things like ``42.0_prt`` if you like to have another value than zero.
```

Comment:
Maybe a word is missing here: "code", "use"?
```suggestion
You can also code things like ``42.0_prt`` if you like to have another value than zero.
```

Reply:
Oh yes, this sentence no verb
```suggestion
You can also write things like ``42.0_prt`` if you like to have another value than zero.
```